app/assets/javascripts/member-facing.js.erb

```diff
@@ -19,6 +19,8 @@
 require("<%= Settings.external_js_path %>");
 <% end %>
 
+window.URI = require('urijs');
```

Comment:
In that case the better approach is to import it in the file where you're using it - if that's possible. If you're using it in a template where you don't have access to `require`, then I can understand why it's made a global.